_pages/tinypinch.yaml

```diff
@@ -0,0 +1,5 @@
+ ---
+ githubHandle: YOUR-USERNAME
+ pageUrl: YOUR-USERNAME.github.io
+ timestamp: TODAYS-DATE-YYYY-MM-DD
+ ---
```

Comment:
@tinyPinch you'll need to actually replace the contents of the file with your own information. Let us know if we can help!